baybe/recommenders/pure/bayesian/botorch.py

```diff
@@ -216,7 +216,7 @@ def _recommend_hybrid(
         from botorch.optim import optimize_acqf_mixed
 
         # Transform discrete candidates
-        candidates_comp = self.surrogate_model.transform_inputs(candidates_exp)
+        candidates_comp = searchspace.transform(candidates_exp, allow_missing=True)
```

Comment:
Why do we allow missing here?

Reply:
It's needed because I called the transform method of the seachspace but pass only discrete columns (see comment above the line of code). But your question made me realize that it's much better to simply call `searchspace.discrete.transform` instead, which doesn't require activating the exception handling mechanism! Changed it 👍🏼